custom_components/tuya_local/devices/sunred_smart_triangle_heater.yaml

```diff
@@ -0,0 +1,109 @@
+name: heater
+products:
+  - id: f2sycol9awtabrsp
+    manufacturer: Sunred
+    model: ZHQ2582
+    name: SMART Triangle Dark Wall Model
+entities:
+  - entity: switch
+    dps:
+      - id: 1
+        type: boolean
+        name: switch
+  - entity: sensor
+    class: temperature
+    dps:
+      - id: 3
+        name: sensor
+        type: integer
+        unit: C
+        class: measurement
+        readonly: true
+  - entity: select
```

Comment:
I think this would be better converted to a number entity, so a slider can be used. The type of the dp needs to remain string, but the `value` attributes of the mapping should have the quotes removed.

Reply:
Is the slider working as expected?

Reply:
Added range in last commit, now I'm able to turn it on and set the level correctly